examples/yaml-highlight/yaml_hl.py

```diff
@@ -2,7 +2,7 @@
 
 import yaml, codecs, sys, os.path, optparse
 
-class Style:
+class Style(object):
```

Comment:
You can add
```python
__metaclass__ = type
```
in the beginning of all modules, which will enforce new-style classes by default. And this way you don't need to explicitly inherit `object` at all.